### ISSUE TITLE
libwebkit2gtk-4.0 support?

### ISSUE THREAD

**Issue by paultag** [sourcegraph/go-webkit2#31]
I was just playing around with pulling down HTML and rendering it, and I came across this library - thanks for all your hard work, looks really great!

It looks like libwebkit2gtk-3.0 was removed from Debian testing and unstable, which means the next Ubuntus will have that package missing as well.

I'm super happy to test any changes -- unfortunately, I don't know much GTK otherwise I'd have set in a PR. Thanks for putting up with a basically unhelpful bug.

I'll glance through the GTK changelog and see how bad it's going to be, but I seem to recall some angry people on the internet saying it was basically totally API incompatible. Woof.

**Comment by paultag** [comment]
After asking around, this isn't actually the famed GTK4, so it's just that webkit2 has a new bump. I'll check into it.

**Comment by paultag** [comment]
```diff
-// #cgo pkg-config: webkit2gtk-3.0
+// #cgo pkg-config: webkit2gtk-4.0
```

Actually appears to compile. I'll see if it's happy

**Comment by paultag** [comment]
Yep, that plus https://github.com/sourcegraph/go-webkit2/pull/32 allowed me to build the example!

**Comment by sqs** [comment]
Merged sourcegraph/go-webkit2#32.

@paultag can you please submit a PR with that diff to use 4.0 if that is necessary? Thanks!

**Comment by paultag** [comment]
Will do! I'll also send another PR for the user data changes as well.

Thanks!

**Comment by paultag** [comment]
Closed in #34
